pyvista/plotting/charts.py

```diff
@@ -1374,40 +1371,36 @@ def show(
 
         Parameters
         ----------
-        off_screen : bool, optional
+        off_screen : bool, default: :attr:`pyvista.global_theme.off_screen
+            <pyvista.themes.DefaultTheme.off_screen>`
```

Comment:
```suggestion
        off_screen : bool, default: pyvista.global_theme.off_screen
```
I believe the warnings in the doc build are caused by these attribute defaults. Based on the other defaults, I assume this has to be 'plain python' and no sphinx references. Not sure if my suggestion would work though or how it looks in the final docs (because it would be nice to keep a linkable reference to these defaults).
Once we know how to make it work, it might be worthwhile to also change it accordingly in the `show` method of `Plotter` (as this docstring was copied from there).